package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.3",
+    "@storybook/react": "7.0.0-rc.6",
```

Comment:
Please remove this package. 

If you rebase correctly, you will find everything to do with storybook is where it needs to be.

If you got an error close to something like, `This should be a dependency and not a devDependency` inside your stories, then the issue is, you are trying to import the file, while you should be importing the `type` of the file.

Reference other stories @SergeInGithub and I wrote for context.